src/main/java/io/spokestack/spokestack/tts/SpokestackTTSClient.java

```diff
@@ -2,17 +2,12 @@
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import io.spokestack.spokestack.util.Base64;
+import io.spokestack.spokestack.util.Crypto;
```

Comment:
nice refactoring here, we could even reuse this for the cloud nlu api

Reply:
True; that wouldn't be a bad thing to add to the roadmap. Shouldn't take too long.